examples/divisions/division_area/country_land.yaml

```diff
@@ -304,6 +304,7 @@ properties:
   version: 0
   subtype: country
   class: land
```

Comment:
leaving class land?

Reply:
We chatted about this - we're planning to keep division_area and division_boundary `class` values as is for now and only add `class` for division features. Once area and boundary features are updated, we'll update these.